guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,352 @@
+
+A receiver should be able to set up payjoin during onboarding on the platform (app or POS system) or any time after that. The following is a standalone payjoin setup flow.
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/payjoin-case-study/payjoin-receiver-flow@2x.png"
+   retina = "assets/images/guide/case-studies/payjoin-case-study/payjoin-receiver-flow@2x.png"
+   alt-text = "payjoin receiver flow"
+   width = 800
+   height = 458
+%}
+
+A detailed process flow integrated with onboarding is available in this [Figjam file](https://www.figma.com/file/NzMvwyzP7x5jfGmwNUKRov/PayJoin-Process-Flows?node-id=0%3A1&t=wSwewTOkQddWhagl-1).
+
+##### Requesting payjoin
+
+Once the payjoin setup is functional, all payment requests (BIP-21 payment links or encoded as QRs) should contain the necessary information (including endpoint address) to perform the payjoin transaction. No separate user action should be required.
+
+
+#####  Payjoin Status/Settings
+
+A dedicated section in the settings page might be a good idea to provide dedicated space in the UI to manage Payjoin settings and monitor its status. This would also provide UI space for users using a cold wallet to discover and set up payjoin. This section could contain:
+
+- Payjoin setup workflow
+- Additional payjoin features like third party endpoints, address substitution, cold-wallet backfill
+- Monitor payjoin readiness: hot wallet status, pj-endpoint status etc.
+
+
+## Insights
+
+We learnt many things over the course of this case study, and the user flows we have designed already use many of these insights.
+
+- Payjoins are a great way to consolidate UTXOs so they pay lower fees in the long run – this can be especially useful to exchanges, trading desks who need this the most and possess technical ability to implement a robust setup
+- Due to the back-&-forth between parties which can extend over multiple iterations/rounds, this case study introduces the term ‘payjoin handshake’ to encompass all the events between user actions
+- Maintaining an always-online endpoint seems to be the biggest hurdle for payjoin implementation, we list some alternatives here
+- Payjoin burdens the receiver with more requirements but also provides more benefits and opportunities
+- Address substitution is a powerful idea from BIP-78: it is a risk that can become a powerful ally for the payjoiner
+- Payjoins receivers can leverage their daily spending wallet-saving wallet setup, where the daily spending is used by the receiver as the payjoin hot wallet
+- Payjoin implementations should be mindful of the round-fee-rate heuristic for identifying payjoins, handle it without need for user decisions
+
+
+## New Horizons
+
+Payjoins can be complicated to understand or implement given the multiple parties and moving parts (such as the endpoint) involved, but this also opens the door for multiple features and services.
```

Comment:
```suggestion
Payjoins can be complicated to understand or implement, given the multiple parties and moving parts (such as the endpoint) involved. But, this also opens the door for multiple features and services.
```

Reply:
Agree it is kinda long as a bullet point, but I avoid using periods in bullet points.